spec/workers/mute_worker_spec.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe MuteWorker do
```

Comment:
There were absolutely no tests for MuteWorker, so this is an attempt at adding some.

Reply:
I'm not sure if this is the correct way for us to test MuteWorker.